emr.post_update.php

```diff
@@ -38,5 +38,40 @@ function emr_post_update_00002(&$sandbox) {
       $revision->save();
     }
   }
+}
+
+/**
+ * Track the entity meta default values in the new table.
+ */
+function emr_post_update_00003(&$sandbox) {
```

Comment:
We can consider this as "fixing the API" and move it to a hook_update_N() implementation so that we can drop the field on one go.